### DISCUSSION TITLE
Do not import `@jest/globals` outside of the Jest test environment

### DISCUSSION THREAD

**Discussion by biglebronski** [vuejs/test-utils#290] [Q&A]
Hi,

I see in the docs that mocks are implemented in the global scope during `mount()`. I have some tests that are testing against Typescript classes and modules, not Vue components, and I'm trying to use `jest.mock()` there and getting this error.

`Test suite failed to run`
`Do not import '@jest/globals' outside of the Jest test environment`

This error seems to have started with `babel-jest@^25`, and can be resolved by rolling back that package to `v21.2`. However, there are multiple potential issues with doing that, as I don't want to use different versions of babel/babel-jest/ts-jest and also `vue-test-utils-next` wants `babel-jest@^25.2.3`.

Has anyone here seen this before and possibly know a better way to resolve it?

Here is my jest config if that helps.

```
module.exports = {
  preset: "@vue/cli-plugin-unit-jest/presets/typescript-and-babel",
  collectCoverage: true,
  collectCoverageFrom: ["src/**/*.{ts,tsx}", "!src/**/*.d.ts", "!src/main.ts"],
  coverageReporters: ["text", "cobertura"],
  globals: {
    "ts-jest": {
      babelConfig: true
    }
  },
  transformIgnorePatterns: [
    "/node_modules/(?!(vuetify|@tgcs|roboto-fontface|material-design-icons-iconfont|@mdi))"
  ],
  transform: {
    "^.+\\.(ts|tsx)?$": "ts-jest",
  },
  rootDir: path.resolve(process.cwd(),"."),
  testMatch: [
    "**/tests/unit/*.(ts|tsx)",
    "**/tests/unit/**/*.(ts|tsx)",
    "**/__tests__/*.spec.(ts|tsx)"
  ],
  modulePathIgnorePatterns: [
    "<rootDir>/\.yalc/@tgcs/.*/\.yalc"
  ]
};
```

Thanks in advance.

**Comment by lmiller1990** [comment]
I have not seen that error before. I don't think it is relate to test utils (not clear what you mean by mocks in the global scope messing up things).

You will need to post a full reproduction if you want help, there is no way to reproduce the error with just your jest.config.

**Comment by iliubinskii** [comment]
Most likely you have installed two jest versions because of version conflict.
In this case stack trace should have path with double "node_modules", e.g. .../node_modules/.../node_modules/...
